frontend/src/lib/types/QueueItemType.ts

```diff
@@ -73,9 +74,44 @@ export const isAuto = (type: QueueItemType): boolean => {
 };
 
 /**
- * Get the concrete type for a queue item type,
- * @param type The QueueItemType
- * @returns Factory for AuotFactories, Mine for AutoMines, etc
+ * Check if a {@linkcode ProductionQueueItem} will be fully skipped and entirely unbuildable.
+ * @param item the {@linkcode ProductionQueueItem} to check; must have estimates filled in
+ * @returns `true` if item is fully skipped (nothing will be built for the next 100 years)
+ */
+export const isFullySkipped = (item: ProductionQueueItem) => {
+	return isAuto(item.type) && item.yearsToBuildOne == Infinite && item.yearsToBuildAll == Infinite;
+}
```

Comment:
```suggestion
export const isFullySkipped = (item: ProductionQueueItem) =>  isAuto(item.type) && item.yearsToBuildOne == Infinite && item.yearsToBuildAll == Infinite;
```
if doing a const func that just returns a var, don't wrap it in braces with a return, the return is implied by the arrow func

Reply:
I tend to prefer `function` in javascript unless it's a simple one liner, or if you need to use someone else's type defs (like for input events)